site/assets/scss/_component-examples.scss

```diff
@@ -217,6 +217,42 @@
   }
 }
 
+.bd-example-inset-utils {
+  display: flex;
+  flex-wrap: wrap;
+
+  span {
+    position: absolute;
+    display: block;
+  }
+
+  div {
+    position: relative;
+    width: 64px;
+    height: 64px;
+    margin-right: 12px;
+    margin-bottom: 12px;
+
+    &:nth-child(1) span,
+    &:nth-child(3) span {
+      height: 32px;
+    }
+
+    &:nth-child(2) span,
+    &:nth-child(4) span {
+      width: 32px;
+    }
+
+    &:nth-child(6) span,
+    &:nth-child(7) span,
+    &:nth-child(8) span,
+    &:nth-child(9) span {
+      width: 32px;
+      height: 32px;
```

Comment:
Use `rem` for those examples, it'd be more responsive if a user changes his base font size in his browser settings.

Reply:
done!

Reply:
Did you push these changes?

Reply:
Sorry my bad. I forget send my changes

Reply:
done!